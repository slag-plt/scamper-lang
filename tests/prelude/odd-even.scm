(odd? 2)
(odd? (- 10 9))
(odd? 0.0)
(odd? (- 7 4))
(odd? 8)
(even? 2)
(even? 1)
(even? (- 7 3))
(even? 8)
(even? 0.0)