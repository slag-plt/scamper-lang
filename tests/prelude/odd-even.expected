#f
#t
#f
#t
#f
#t
#f
#t
#t
#t
